quarkus/admin/src/main/resources/application.properties

```diff
@@ -49,3 +49,21 @@ quarkus.index-dependency.guava.group-id=com.google.guava
 quarkus.index-dependency.guava.artifact-id=guava
 quarkus.index-dependency.protobuf.group-id=com.google.protobuf
 quarkus.index-dependency.protobuf.artifact-id=protobuf-java
+
+#quarkus.datasource.db-kind=pgsql
+#quarkus.datasource.jdbc.url=polaris
+#quarkus.datasource.username=polaris
+#quarkus.datasource.password=polaris
+quarkus.datasource.\"realm1\".db-kind=pgsql
```

Comment:
Is this for testing?

Reply:
This is for this behaviour 

```
I observed some unexplainable quarkus behaviour and spent days on debugging it, but now i have some way to work around, with custom DS of quarkus if i don't specify them in application prop and even though i specify it my PostgresLifeCycleManagement test container it doesn't create these beans, so declearing even the garbage in the application prop does the trick.
```

Reply:
having these obviously test-like entries in the production config is not nice. Could you make another attempt at removing them? Maybe you meant to put them under the `test` or `it` Quarkus profiles?